src/client/eppo-client-assignment-details.spec.ts

```diff
@@ -276,17 +289,27 @@ describe('EppoClient get*AssignmentDetails', () => {
               if (!assignmentFn) {
                 throw new Error(`Unknown variation type: ${variationType}`);
               }
-              const assignmentDetails = assignmentFn(
+              const result: IAssignmentDetails<Variation['value'] | object> = assignmentFn(
                 flag,
                 subject.subjectKey,
                 subject.subjectAttributes,
                 defaultValue,
               );
-              expect(assignmentDetails).toMatchObject({
-                ...subject.assignmentDetails,
+              expect(result.value).toEqual(subject.assignment);
+              expect(result.evaluationDetails).toEqual({
+                ...subject.evaluationDetails,
+                environmentName: 'Test',
                 configFetchedAt: expect.any(String),
                 configPublishedAt: expect.any(String),
               });
+              expect(result.evaluationDetails.configPublishedAt).toEqual(
+                '2024-04-17T19:40:53.716Z',
+              );
+              // date fetched will always be current day when running this test
```

Comment:
Unless we cross the UFC boundary 😅 

You could just check that it is set to a date after the test start. See an example [here](https://github.com/Eppo-exp/eppo/blob/main/packages/api/src/user/resolvers/user.resolver.e2e-spec.ts#L43).

Reply:
Nice, I like that approach. I'll update.

Reply:
Updated. Didn't want to bring in jest-extended for this, but I used the same approach.